interoperability-tests/Testing-between-Gramine-and-IntelSGXSDK-ratls.md

```diff
@@ -0,0 +1,158 @@
+Testing between Gramine and rats-tls
+====
+
+# Introduction
+
+This document records the interoperability testing between [Gramine](https://github.com/gramineproject/gramine) and [Intel SGXSDK](https://github.com/intel/linux-sgx), both of which have implemented Interoperable RA-TLS.
+
+We conducted two sets of tests, in which Gramine and Intel SGXSDK ra-tls respectively verified each other's certificates and established TLS secure channels. You can verify our tests by following the steps below.
+
+# Prerequisites
+
+## Intel SGXSDK 
+
+To run the tests, first you need to download and install Intel SGXSDK and corresponding PSW packages on your system. 
+
+Interoperable RA-TLS was implemented since Intel SGXSDK 2.22 release. 
+Download the software package from following link:
```

Comment:
`from following` -> `from the following`

Reply:
updated